webapp/client/socket.ts

```diff
@@ -0,0 +1,73 @@
+import types from '@/bindings/bindings'
+
+export type ServerCallback = (socket:Socket) => void
+export type OnMessageCallback = (message:types.ServerMessage, socket:Socket) => void
+
+export interface UpdateObject {
```

Comment:
What is this? Add a comment explaining what is an `UpdateObject`.

Reply:
ok